src/build/leo/aleo-sdk/js-ts/guide.md

```diff
@@ -1 +1,1187 @@
+        subgraph ProgramManager
+            subgraph Aleo-Wasm-Module
+                in-memory-program
+            end
+        end
+    end
+
+    p1-->p2--load program--oin-memory-program-.ZK result.->user
+    user-.user input.->in-memory-program
+    in-memory-program-."ZK result (Optional)".->Aleo-Network
+```
+
+### 2.3 WebAssembly Initialization
+
+❗WebAssembly must be initialized before calling any SDK functions. The current Aleo SDK manages the wasm initialization. Therefore, the workers must be defined properly.
+
+Aleo programs are made zero knowledge through the usage of `ZkSnarks`. The Rust code behind Aleo programs and the ZkSnarks
+that make them zero knowledge are hosted in the [snarkVM Repository](https://github.com/AleoHQ/SnarkVM). The Aleo SDK
+compiles this code to WebAssembly and creates JavaScript bindings, enabling Aleo programs to run directly in the browser.
+
+Before any logic within the SDK is run within the browser however, the WebAssembly module the SDK contains must be
+initialized before any SDK functions can be executed. This is done simply by calling the `initializeWasm` function at a
+point in your code before any other SDK functions are called:
+```typescript
+import { Account, initializeWasm } from '@aleohq/sdk';
+
+// Assuming top-level await is enabled. This can also be initialized within a promise.
+await initializeWasm();
+
+/// Create a new Aleo account
+const account = new Account();
+````
+
+An example of how to initialize WebAssembly in a React app is shown in [Section 2.8](#28-React-Example)
+
+### 2.4 Local Program Execution
+A simple example of running the hello world program within the web browser is shown below:
+```typescript
+import { Account, Program } from '@aleohq/sdk';
+
+/// Create the source for the "hello world" program
+const program = "program helloworld.aleo;\n\nfunction hello:\n    input r0 as u32.public;\n    input r1 as u32.private;\n    add r0 r1 into r2;\n    output r2 as u32.private;\n";
+const programManager = new ProgramManager();
+
+/// Create a temporary account for the execution of the program
+const account = new Account();
+programManager.setAccount(account);
+
+/// Get the response and ensure that the program executed correctly
+const executionResponse = await programManager.executeOffline(program, "hello", ["5u32", "5u32"]);
+const result = executionResponse.getOutputs();
+assert(result === ["10u32"]);
+```
+
+### 2.5 Program execution on the Aleo Network
+The SDK also provides the ability to execute programs and record an encrypted transcript of the execution on the Aleo
+network that anyone can trustlessly verify.
+
+This process can be thought of being executed in three steps:
+1. A program is run locally
+2. A proof that the program was executed correctly and that the outputs follow from the inputs is generated
+3. A transcript of the proof is generated client-side containing encrypted proof data (see [Section 2.6](#4-managing-records-and-private-state))
```

Comment:
Same as section 2.8 link for section 2.6

Reply:
I think it refers to Section 4, so I updated both the link and the section it refers.